src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -199,7 +199,7 @@
           offsetDelay = index * delay;
 
         styles.opacity = ctrl.isOpen ? 1 : 0;
-        styles.transform = styles.webkitTransform = ctrl.isOpen ? 'scale(1)' : 'scale(0)';
+        styles.transform = styles.webkitTransform = ctrl.isOpen ? 'scale(1)' : 'scale(0.1)';
```

Comment:
Just curious - why `0.1` instead of `0`.  Maybe include a comment explaining this.